appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/GalleryList.java

```diff
@@ -468,27 +468,18 @@ private void refreshApps(GalleryAppListResult appsResult, int requestId, boolean
   }
 
   /**
-   * Gets the number of projects
+   * Gets the number of selected apps
    *
-   * @return the number of projects
-   */
-  public int getNumProjects() {
-    return apps.size();
-  }
-
-  /**
-   * Gets the number of selected projects
-   *
-   * @return the number of selected projects
+   * @return the number of selected apps
    */
   public int getNumSelectedApps() {
```

Comment:
According to IntelliJ, `getNumSelectedApps`, `getSelectedApps`, and `setSelectedTabIndex` are also not called on `master`, and it looks like they are not referenced on the branding branch either.

Reply:
@ewpatton I believe these functions are for future work... Let's leave them for now.

Reply:
@jisqyv Sounds good to me.